src/modules/helpchan.ts

```diff
@@ -39,6 +42,20 @@ This channel will be dedicated to answering your question only. Others will try
 For more tips, check out StackOverflow's guide on **[asking good questions](https://stackoverflow.com/help/how-to-ask)**.
 `;
 
+const occupiedMessage = (asker: User) => `
+**This channel is claimed by ${asker.toString()}.**
```

Comment:
You can probably drop the explicit `toString`.

Reply:
Done